docs/docs/networking.md

```diff
@@ -4,10 +4,13 @@ title: Networking
 
 # Networking
 
-Marathon continues to support the [legacy ports API](ports.html) that was shipped in versions 1.4.x and prior.
-All new applications should be declared using the new, non-deprecated networking API fields that are documented here.
-Applications using the old networking API fields will be automatically migrated to the new networking API in Marathon 1.5.x.
-Support for deprecated networking API fields will be dropped completely in some future release according to Marathon's official API deprecation policy [MARATHON-7165](https://jira.mesosphere.com/browse/MARATHON-7165).
+This document describes the networking API released as of Marathon 1.5.
+
+While Marathon continues to support the [legacy ports API](ports.html) that was shipped in versions 1.4.x and prior, all new applications should be declared using the new, non-deprecated networking API fields that are documented here.
```

Comment:
"continues to support" is probably too generous. Marathon continues to "consume" the old 1.4 compatible format but it only emits the new 1.5 format